### PR TITLE
Feature/86 update to protocol v2

### PR DESCRIPTION
**Pull request by cmcWebCode40**
- [x] Updated verida protocol libraries to `v2.0.1`

### CODE REVIEW

File: components/account/src/lib-components/vda-account.vue [hunk 1]
```diff
@@ -207,6 +212,8 @@ export default /*#__PURE__*/ defineComponent({
     },
     async init() {
       if (VeridaHelper.hasSession(this.contextName) && !VeridaHelper.context) {
+        console.log(VeridaHelper.hasSession(this.contextName));
```
Comment (aurelticot) [inline components/account/src/lib-components/vda-account.vue:215]: You want to keep the console.log?

File: components/account/package.json [hunk 2]
```diff
@@ -1,81 +1,81 @@
...
+    "publish": "yarn build && lerna publish",
+    "format:check": "npx prettier src/ --check",
+    "format": "npx prettier src/ --write",
+    "lint": "vue-cli-service lint"
+  },
+  "dependencies": {
+    "@typescript-eslint/eslint-plugin": "^5.38.0",
+    "@typescript-eslint/parser": "^5.38.0",
+    "@verida/account-web-vault": "^2.0.1",
+    "@verida/client-ts": "^2.0.1",
```
Comment (aurelticot) [inline components/account/package.json:34]: We were discussing of moving the two Verida packages into `peerDependencies`

**Review by aurelticot — CHANGES_REQUESTED** [review]

File: components/account/package.json (hunk 2 above)
Comment (cmcWebCode40) [reply components/account/package.json:34]: ok, how do I go about it?,   i haven't tried it before

File: components/account/src/lib-components/vda-account.vue (hunk 1 above)
Comment (cmcWebCode40) [reply components/account/src/lib-components/vda-account.vue:215]: Removed.
no, i will set up eslint + prettier after the release.

File: components/account/package.json (hunk 2 above)
Comment (aurelticot) [reply components/account/package.json:34]: In package.json, move them under `peerDependencies`:

```json
  "peerDependencies": {
    "@verida/account-web-vault": "^2.0.1",
    "@verida/client-ts": "^2.0.1",
    "vue": "^3.0.5"
  },
```

It means, these packages will have to be installed (manually) by all apps using this verida-vue-component package. Similar to Vue.

Similar to Vue, as well, if you need them for testing, you can add them to the `devDependencies` as well.

Comment (cmcWebCode40) [reply components/account/package.json:34]: thanks

**Review by aurelticot — APPROVED** [review]

**Pull request merged**